website/project/metadata/osf-preregistration.json

```diff
@@ -0,0 +1,378 @@
+            "type": "string",
+            "format": "textarea",
+            "title": "Sample size rationale",
+            "description": "This could include a power analysis or an arbitrary constraint such as time, money, or personnel.",
+            "help": "We used the software program G*Power to conduct a power analysis. Our goal was to obtain .95 power to detect a medium effect size of .25 at the standard .05 alpha error probability. <br /><br /> How responses will be evaluated: This gives you an opportunity to specifically state how the sample size will be determined. A wide range of possible answers is acceptable; remember that transparency is more important than principled justifications. If you state any reason for a sample size upfront, it is better than stating no reason and leaving the reader to “fill in the blanks.” Acceptable rationales include: a power analysis, an arbitrary number of subjects, or a number based on time or monetary constraints."
+        }, {
+            "qid": "q10",
+            "nav": "Stopping rule",
+            "title": "Stopping rule",
+            "type": "string",
+            "format": "textarea",
+            "description": "If your data collection procedures do not give you full control over your exact sample size, specify how you will decide when to terminate your data collection. ",
+            "help": "We will post participant sign-up slots by week on the preceding Friday night, with 20 spots posted per week. We will post 20 new slots each week if, on that Friday night, we are below 320 participants. <br /><br /> How responses will be evaluated: You may specify a stopping rule based on p-values only in the specific case of sequential analyses with pre-specified checkpoints, alphas levels, and stopping rules. Unacceptable rationales include stopping based on p-values if checkpoints and stopping rules are not specified. If you have control over your sample size, then including a stopping rule is not necessary, though it must be clear in this question or a previous question how an exact sample size is attained."
+        }]
+    }, {
+        "id": "page4",
+        "title": "Variables",
+        "description": "In this section you can describe all variables (both manipulated and measured variables) that will later be used in your confirmatory analysis plan. In your analysis plan, you will have the opportunity to describe how each variable will be used. If you have variables which you are measuring for exploratory analyses, you are not required to list them, though you are permitted to do so.",
+        "questions": [{
+            "qid": "q11",
+            "nav": "Manipulated",
+            "type": "object",
+            "title": "Manipulated variables",
+            "description": "Describe all variables you plan to manipulate and the levels or treatment arms of each variable. For observational studies and meta-analyses, simply state that this is not applicable.",
```

Comment:
Should be: "Describe all variables you plan to manipulate and the levels or treatment arms of each variable. This is not applicable to any observational study."